x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/screens/home/index.tsx

```diff
@@ -114,7 +114,11 @@ function InstalledPackages() {
 
 function AvailablePackages() {
   useBreadcrumbs('integrations_all');
-  const [selectedCategory, setSelectedCategory] = useState('');
+  const history = useHistory();
+  const queryParams = new URLSearchParams(useLocation().search);
+  const initialCategory =
+    queryParams.get('category') !== null ? (queryParams.get('category') as string) : '';
```

Comment:
Is `null` significant? If possible, I'd prefer
```suggestion
const initialCategory = queryParams.get('category') || '';
```

I find it more straightforward and we don't lose any safety, afaict.